.gitignore

```diff
@@ -1,13 +1,16 @@
+# ingore local env files
+.env.local
+
 *.pb.cc
 *.pb.h
 *.so
 *.swp
-.idea
-.vscode
+
 CMakeCache.txt
 CMakeFiles
 CTest*.cmake
 Makefile
+.conan
```

Comment:
Why this .conan line?
I know only two cases for this folder to be created:
* root case: it is created in /root/.conan
* user case: it is created in /home/USER/.conan

Do you hace centreon-collect at the same place?

Reply:
It came from the backport. Originaly it has been added here : https://github.com/centreon/centreon-collect-gha/blob/develop/.gitignore#L13

Do not hesitate to suggest directives removal ;)